### PR TITLE
Updated Embed Media - Audio, Photos, Videos page

### PR DESCRIPTION
**Pull request by candideu**
Added easier method of embedding video URLs, and added tips for Hiccup markup for HTML video and audio attributes

### CODE REVIEW

**Review by cnrpman — APPROVED** [review]
LGTM

File: pages/Embed Media - Audio, Photos, Videos.md [hunk 1]
```diff
@@ -60,10 +66,13 @@
 				  https://logseq.github.io/screenshots/1.png
 				  ```
 				  #+BEGIN_TIP
-				  	  You can specify the image size by formatting the image link as such: `![](https://logseq.github.io/screenshots/1.png){:height 469, :width 819}`
+				  You can specify the image size by formatting the image link as such: `![](https://logseq.github.io/screenshots/1.png){:height 469, :width 819}`
 				   #+END_TIP
 - ## Videos
-	- Using [[Hiccup]] and certain slash commands, it is possible to embed video to your page and play it back.
+	- You can also easily embed and play back video files by using the `{{video }}` command, and pasting the link to the file inside the brackets after `video`:
```
Comment (logseq-cldwalker) [inline pages/Embed Media - Audio, Photos, Videos.md:72]: `{{ video}}` is a macro but I don't mind fixing wording

**Review by logseq-cldwalker — APPROVED** [review]
@candideu Thanks for the improvements and fixes! 👍

File: pages/Embed Media - Audio, Photos, Videos.md (hunk 1 above)
Comment (candideu) [reply pages/Embed Media - Audio, Photos, Videos.md:72]: Yes, feel free to correct the wording! Thanks for flagging this :)

**Pull request merged**